source/manual/alerts/low-available-disk-space.html.md

```diff
@@ -59,6 +59,35 @@ This command will take a while to complete, depending - of course - on
 how large your two starting folders are. After it has completed, push
 Nagios to re-check the service on each affected host.
 
+## Low available disc space on /mnt/crawler_worker on mirrorer
+
+This can be caused when the AWS autoscaler re-spins a VM and the newly created
+VM does not attach the secondary EBS volume properly.
+
+### Cause
+An EBS volume can only be attached to one VM at a time.  When the autoscaler
+terminates a VM, AWS disconnects the EBS volume and releases a lock, whilst at
+the same time also spinning up the replacement VM.  The new VM can spin up and be
+provisioned faster than the lock can be released.
+
+### Solution
+Manually re-attach the EBS volume tagged with mirrorer to the mirrorer
+instance, then run puppet on the instance. This will cause puppet to correctly
+mount the EBS volume in /mnt/crawler_worker.
+
+1. Login to https://gds-users.signin.aws.amazon.com/console
+1. Switch role to the appropriate environment: Integration, Staging or Production
+1. Click Services -> EC2
+1. Change region to "Europe (Ireland) eu-west-1"
```

Comment:
Interesting: do you need to do this? (I've never had to do this.)

Reply:
It depends on the user and whether they use regions.